src/borg/remote.py

```diff
@@ -535,14 +536,15 @@ def __init__(self, location, create=False, exclusive=False, lock_wait=None, lock
         self.msgid = 0
         self.rx_bytes = 0
         self.tx_bytes = 0
-        self.to_send = b''
+        self.to_send = EfficientCollectionQueue(1024 * 1024, lambda: b'')
```

Comment:
instead of that lambda, you could give just `member_type=bytes`.

usually, calling `member_type()` will just result in an empty instance.